graspologic/layouts/auto.py

```diff
@@ -260,7 +256,14 @@ def _node_positions_from(
     sizes = _compute_sizes(degree)
     covered_area = _covered_size(sizes)
     scaled_points = _scale_points(down_projection_2d, covered_area)
-    partitions = leiden(graph, random_seed=random_seed)
+    if isinstance(graph, nx.DiGraph):
+        temp_graph = symmetrize(graph)
```

Comment:
hey @dlee0156 have you tested this line when the input is a networkx graph? i'm worried this line will break

also, in general, we should probably add at least one test to our test suite where the input is directed. let me know if you need help getting started with adding/running tests?

Reply:
@bryantower - so @bdpedigo and I have talked before about automatic symmetrization.  our go-to most of the time is to simply sum the edges, whereas I think symmetrize and JHU's approach has been to average them.  I REALLY don't think it matters, but I wanted yet-another sanity check, mostly just to make me feel better.

@dlee0156 I just checked, and symmetrize only works on adjacency matrices, so you'll need to write your own `give me an undirected graph form of this digraph` function (I'm just now wondering if this is something that already exists in nx - if not, we should probably put it in a `utils.networkx.directed_to_undirected()` function or something)

Reply:
https://networkx.org/documentation/stable/reference/classes/generated/networkx.DiGraph.to_undirected.html#networkx.DiGraph.to_undirected is, in theory, what we want - but we specifically do NOT want that approach for edge weight selection, which is probably one of the worst defaults I can imagine

Reply:
@bdpedigo Yep it works for a networkx graph and yes, that would be super helpful thank you!

@daxpryce Would something like this be sufficient or should the function add / average the edges without converting it back and forth from matrix to graph?
```python
def _to_undirected(graph: nx.DiGraph) -> nx.Graph:
    adj = nx.to_numpy_matrix(graph)
    sym_adj = symmetrize(adj)
    G = nx.Graph(sym_adj)
    return G
```

Reply:
i think we'd want to do this conversion in networkx internally. something like this https://docs.neurodata.io/notebooks/pedigo/graspologic/connectome/2021/06/03/hemibrain-hierarchy.html#Make-the-network-undirected

Reply:
one comment about the above code is that it doesn't preserve metadata - if we are exposing this as a utility function, maybe it should? but then there are weird cases, like what if an i -> j and a j -> i edge have differing metadata?

Reply:
just made changes to `_to_undirected` so that it keeps it in networkx. For this function though, how should we deal with graphs that have unweighted edges?